docker-compose.yml

```diff
@@ -49,9 +54,11 @@ services:
         container_name: notification_service-api
 
 volumes:
+    djangodata:
+        driver: local
     pgdata:
         driver: local
 
-networks:
-    default:
-        name: helsinki
+#networks:
+#    helsinki:
+#        driver: bridge
```

Comment:
The old configuration is used pretty much in every project we have:
```
networks:
    default:
        name: helsinki
```
Is there a reason why this should not use it like that?

Reply:
There were host resolving issues using podman, and this resolved that issues as well as running docker locally on VM rather than direct on "localhost". We have similar setups on other projects too.